webapp/Makefile

```diff
@@ -12,16 +12,18 @@ SUBDIRS = config
 copy-bundle-assets:
 # We can not use bin/console here, as when using a fakeroot,
 # the include paths are broken. We just copy in the data we need
-	-rm -rf public/bundles/nelmioapidoc
+	-rm -rf public/bundles/nelmioapidoc/*
```

Comment:
Nitpick: why not delete the whole directory? It gets recreated below, and this is/should be run as the normal user (because it's part of the `domserver` build target), so it guarantees a clean copy.

Reply:
I've tried that but it doesn't work. I think because we have this in the main Makefile (at the root):
`install-domserver: domserver composer-dump-autoload domserver-create-dirs` so we remove the folder again as super user (if you use sudo) and it would be owned by `root:root`
```
ls -atrl webapp/public/bundles/nelmioapidoc/
total 36
drwxr-xr-x 2 domjudge domjudge 4096 Jun 16  2023 swagger-ui
-rw-r--r-- 1 domjudge domjudge 6027 Jun 16  2023 style.css
-rw-r--r-- 1 domjudge domjudge 9212 Jun 16  2023 logo.png
-rw-r--r-- 1 domjudge domjudge 1628 Jun 16  2023 init-swagger-ui.js
drwxrwxrwx 3 domjudge domjudge 4096 Feb 18 00:10 ..
drwxr-xr-x 3 root     root     4096 Feb 18 00:10 .
```

So the folder is now owned by root, but the files inside the dir by the current user (domjudge in my setup) or the user used for `make domserver`. Not sure if we can solve this in a nice way or if the `install-domserver` should not run `domserver` at all?

Looking at: https://github.com/search?q=repo%3ADOMjudge%2Fdomjudge%20domserver%3A&type=code we use it mostly to make sure that `make domserver` runs if the user forgot. Removing that first one seems to work but I'm not sure if this doesn't create other behaviour.